### PR TITLE
chore: remove old deploy, rename new-deploy

### PR DESCRIPTION
**Pull request by Imti**
## Motivation

Old deploy no longer being used, new-deploy is being used

## Solution

Remove deploy, rename new-deploy to deploy

## PR Checklist

- [ ] Added Tests
- [ ] Updated Documentation
- [x] Updated CHANGELOG.md for the appropriate package

### CODE REVIEW

File: packages/contracts-bridge/hardhat.config.ts [hunk 1]
```diff
@@ -2,23 +2,24 @@ import "hardhat-gas-reporter";
 import "@nomiclabs/hardhat-etherscan";
 import "hardhat-packager";
 
-import { task } from "hardhat/config";
-import { verifyBridgeDeploy } from "@nomad-xyz/deploy/src/verification/verifyDeploy";
+// import { task } from "hardhat/config";
```
Comment (anna-carroll) [inline packages/contracts-bridge/hardhat.config.ts:5]: you can just delete this!

File: packages/contracts-core/hardhat.config.ts [hunk 2]
```diff
@@ -2,23 +2,24 @@ import "hardhat-gas-reporter";
 import "@nomiclabs/hardhat-etherscan";
 import "hardhat-packager";
 
-import { task } from "hardhat/config";
-import { verifyCoreDeploy } from "@nomad-xyz/deploy/src/verification/verifyDeploy";
+// import { task } from "hardhat/config";
```
Comment (anna-carroll) [inline packages/contracts-core/hardhat.config.ts:5]: ditto, delete!

File: packages/local-environment/src/network.ts [hunk 3]
```diff
@@ -5,6 +5,7 @@ import { Key } from ".";
 import { sleep } from "./utils";
 import { DockerizedActor } from "./actors";
 import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+// TODO: the old deploy (what is currently published to npm) has been deprecated
```
Comment (anna-carroll) [inline packages/local-environment/src/network.ts:8]: @kekonen can we merge this without these TODOs done, or do they need to be addressed?

**Review by anna-carroll — COMMENTED** [review]
looks good!!

**Comment by anna-carroll** [comment]
I think this is the only thing in the old deploy that's not in the new deploy which we use: https://github.com/nomad-xyz/monorepo/issues/258

but we can merge / delete old deploy without it

File: packages/local-environment/src/network.ts (hunk 3 above)
Comment (Imti) [reply packages/local-environment/src/network.ts:8]: local-environment seems to be out of date anyways. Would love to delete it in a separate PR

Comment (Imti) [reply packages/local-environment/src/network.ts:8]: PR for deleting local-environment (https://github.com/nomad-xyz/monorepo/pull/266)

**Review by yourbuddyconner — APPROVED** [review]

**Pull request merged**